### ISSUE TITLE
LINUX_SOURCEKIT_LIB_PATH on GitHub Actions Ubuntu 18.04

### ISSUE THREAD

**Issue by valerianb** [jpsim/SourceKitten#648]
Hello,

LINUX_SOURCEKIT_LIB_PATH needs to be set to `/usr/share/swift/usr/lib` on GitHub's `ubuntu-18.04` for Actions.
Is it ok to make a PR to add this to the search paths on Linux ?

Thanks

**Comment by Cyberbeni** [comment]
If you are using a tool from another repo (like SwiftLint), then you can use this action to take care of caching and setting LINUX_SOURCEKIT_LIB_PATH for you: https://github.com/Cyberbeni/install-swift-tool

**Comment by jpsim** [comment]
> Is it ok to make a PR to add this to the search paths on Linux ?

Yes can you please! `install-swift-tool` is nice but it'd be better if this worked with common configurations out of the box too!
